src/fsharp/AttributeChecking.fs

```diff
@@ -267,9 +266,16 @@ let private CheckILAttributes (g: TcGlobals) isByrefLikeTyconRef cattrs m =
 
 /// Check F# attributes for 'ObsoleteAttribute', 'CompilerMessageAttribute' and 'ExperimentalAttribute',
 /// returning errors and warnings as data
-let CheckFSharpAttributes g attribs m = 
-    if isNil attribs then CompleteD 
-    else 
+let langVersionPrefix = "[--langversion:"
```

Comment:
What's this used for?

Reply:
I imagine this is what is used to check for specific text in the attribute. @KevinRansom might still be working on it.

Reply:
Just what @tihan said.   If the attribute is Experimental and it starts with the text "[--langversion:" then don't warn, otherwise behave like any other experimental attribute.